src/assets/content/dictionary/blog.md

```diff
@@ -1,6 +1,12 @@
 ---
 title: blog
-definition: ''
-perspectives: []
+definition: a website that compiles a writer's or group of writers' personal experiences, observations, thoughts into text articles, often these posts have images or links to other websites.
+
+sources:
+- sourceurl: https://www.dictionary.com/browse/blog
+perspectives: 
+- meaning: "blog" can refer to the entire website OR just a single post within that website.
+  role: writer
```

Comment:
Please take a look at our [CONTRIBUTING.md](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md) for the proper formatting of the perspective, note how it is different from the definition. You will also need to take a look at the[ testing instructions](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md#how-do-i-test-my-contribution) there as well. It looks like the Netlify build is failing - possibly from the quotes in your meaning. Try removing the quotes and testing locally or via the Netlify preview.